std/os.zig

```diff
@@ -133,6 +133,11 @@ fn getRandomBytesDevURandom(buf: []u8) !void {
     const fd = try openC(c"/dev/urandom", O_RDONLY | O_CLOEXEC, 0);
     defer close(fd);
 
+    const st = try fstat(fd);
+    if (!system.S_ISCHR(st.mode)) {
+        return error.Unexpected;
```

Comment:
Quoting the docs for the `Unexpected` error:

```zig
    /// The Operating System returned an undocumented error code.
    /// This error is in theory not possible, but it would be better
    /// to handle this error than to invoke undefined behavior.
```

Let's come up with an appropriate name for this error. We could either use one of the error codes from `OpenError` such as `FileNotFound` or `NoDevice`, or add a unique error code here if it is desirable to handle this situation differently. My suggestion follows:

```suggestion
        return error.NoDevice;
```